test/lib/samson/syslog_formatters_test.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require_relative '../../test_helper'
+
+SingleCov.covered!
+
+describe Syslog::Formatters do
+  describe '.json' do
+    it 'returns a json formatted log' do
+      output = StringIO.new
+      logger = Logger.new(output)
+      logger.formatter = Syslog::Formatters::Json.new
+      logger.info('test')
+      output.string.must_equal "{:severity=>\"INFO\", :time=>#{Time.now}, :message=>\"test\"}"
```

Comment:
that's not json :)
... need to add a .to_json ?

Reply:
This is because I have used 'StringIO' as Logger input, I tried 'STDOUT' as Logger input, but can't able to capture output.